modules/catalog-magento/mappers/ToProduct.ts

```diff
@@ -21,7 +21,9 @@ export default (data: any) => ({
   breadcrumbs: [],
   available: (data.stock_status === 'IN_STOCK' ?? false) as boolean,
   meta: {
-    description: data.description?.html ?? '',
+    metaTitle: data.meta_title ?? '',
```

Comment:
Fields inside meta object dont need to have the meta prefix (meta.title, meta.description, ... is enought).